chapter08/package/airflow-movielens/src/airflow_movielens/hooks.py

```diff
@@ -20,7 +20,7 @@ class MovielensHook(BaseHook):
     """
 
```

Comment:
Using this complains about the import of the base hook being deprecated. It differs from the implementation in the dags folder. Please compare the dag folder implementations of the hook, sensor, operator with the ones in the package directory